frontend/public/plugins.ts

```diff
@@ -9,22 +24,18 @@ const activePlugins =
     ? (require('@console/active-plugins').default as ActivePlugin[])
     : [];
 
-export const pluginStore = new PluginStore(
-  activePlugins,
-  new Set(window.SERVER_FLAGS.consolePlugins),
-);
+const dynamicPluginNames = getAllowedDynamicPluginNames();
+
+export const pluginStore = new PluginStore(activePlugins, dynamicPluginNames);
 
 if (process.env.NODE_ENV !== 'production') {
   // Expose Console plugin store for debugging
   window.pluginStore = pluginStore;
-
-  // Expose function to load dynamic plugins directly from URLs
-  window.loadPluginFromURL = loadPluginFromURL;
```

Comment:
Note to self, also remove `SERVER_FLAGS.loadPluginFromURL` in `@types/console/index.d.ts`